src/blockdata/transaction.rs

```diff
@@ -34,8 +34,30 @@ use blockdata::constants::WITNESS_SCALE_FACTOR;
 #[cfg(feature="bitcoinconsensus")] use blockdata::script;
 use blockdata::script::Script;
 use consensus::{encode, Decodable, Encodable};
-use hash_types::*;
-use VarInt;
+use consensus::encode::VarInt;
+use util::amount::Amount;
+use hash_types::{Txid, Wtxid, SigHash};
+
+/// Errors that may be arisen during transaction-specific computations and evaluations.
+#[derive(Debug)]
+pub enum Error {
```

Comment:
It seems like none of these errors are actually used anymore except `InvalidTransaction` which is really non-descriptive.

Reply:
Should I change return type of `get_fee` fn to `Option<Amount>` then?